packages/multiselect/src/css/index.css

```diff
@@ -77,9 +77,10 @@
 
 .psds-multi-select__input-container {
   margin: calc(4px / 2);
+  flex: 1;
 }
 
-.psds-multi-select__input {
+.psds-multi-select__input-container>.psds-field__input {
```

Comment:
nit: I'd put spaces around the child combinator, stylistically treating it like other operators. I'm surprised prettier didn't do this. But then, maybe prettier isn't running on our css.